control/discovery.py

```diff
@@ -882,7 +882,7 @@ def store_async(self, conn, data, cmd_id):
         self_conn.recv_async = True
         self_conn.async_cmd_id = cmd_id
 
-    def _state_notify_update(self, update, is_add_req):
+    def _state_notify_update(self, update, is_add_req, existing_entry_changed):
```

Comment:
@gbregman Could we change the type of is_add_req to an enum to minimize changes to the state notify interface?
```python
from enum import Enum

class StateChangeType(Enum):
    ADD = 1
    CHANGE = 2
    REMOVE = 3
```

Reply:
I preferred to lave it as it was. It doesn't seem that important to me.